Build failure: beets

## Steps To Reproduce

Steps to reproduce the behavior:

1. build beets with the buckets plugin _not_ manually disabled

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

```
Sourcing python-remove-tests-dir-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing python-remove-bin-bytecode-hook.sh
Sourcing pypa-build-hook
Using pypaBuildPhase
Sourcing python-runtime-deps-check-hook
Using pythonRuntimeDepsCheckHook
Sourcing pypa-install-hook
Using pypaInstallPhase
Sourcing python-imports-check-hook.sh
Using pythonImportsCheckPhase
Sourcing python-namespaces-hook
Sourcing python-catch-conflicts-hook.sh
Sourcing sphinx-hook
Sourcing pytest-check-hook
Using pytestCheckPhase
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/zqlbkd0x46i78zfj7gaqk7kqgkdp2lb4-source
source root is source
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file source/test/testall.py
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
applying patch /nix/store/x4ax5s5b2wsrrn26s2xdjkf1k5fniw43-bash-completion-always-print.patch
patching file beets/ui/commands.py
Hunk #1 succeeded at 2417 (offset 36 lines).
patching file test/test_ui.py
Hunk #1 succeeded at 1414 (offset -20 lines).
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Executing pypaBuildPhase
Creating a wheel...
pypa build flags: --no-isolation --outdir dist/ --wheel
* Getting build dependencies for wheel...
* Building wheel...
Successfully built beets-2.1.0-py3-none-any.whl
Finished creating a wheel...
Finished executing pypaBuildPhase
Running phase: pythonRuntimeDepsCheckHook
@nix { "action": "setPhase", "phase": "pythonRuntimeDepsCheckHook" }
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for beets-2.1.0-py3-none-any.whl
Finished executing pythonRuntimeDepsCheck
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: glibPreInstallPhase
@nix { "action": "setPhase", "phase": "glibPreInstallPhase" }
Running phase: installPhase
@nix { "action": "setPhase", "phase": "installPhase" }
Executing pypaInstallPhase
Successfully installed beets-2.1.0-py3-none-any.whl
Finished executing pypaInstallPhase
Running phase: pythonOutputDistPhase
@nix { "action": "setPhase", "phase": "pythonOutputDistPhase" }
Executing pythonOutputDistPhase
Finished executing pythonOutputDistPhase
Running phase: giDiscoverSelf
@nix { "action": "setPhase", "phase": "giDiscoverSelf" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: giDiscoverSelf
@nix { "action": "setPhase", "phase": "giDiscoverSelf" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: giDiscoverSelf
@nix { "action": "setPhase", "phase": "giDiscoverSelf" }
Running phase: giDiscoverSelf
@nix { "action": "setPhase", "phase": "giDiscoverSelf" }
Running phase: glibPreFixupPhase
@nix { "action": "setPhase", "phase": "glibPreFixupPhase" }
Running phase: fixupPhase
@nix { "action": "setPhase", "phase": "fixupPhase" }
shrinking RPATHs of ELF executables and libraries in /nix/store/53z54syw8gyqa5p7q3a1khhlc5967gxq-beets-2.2.0
checking for references to /build/ in /nix/store/53z54syw8gyqa5p7q3a1khhlc5967gxq-beets-2.2.0...
patching script interpreter paths in /nix/store/53z54syw8gyqa5p7q3a1khhlc5967gxq-beets-2.2.0
stripping (with command strip and flags -S -p) in  /nix/store/53z54syw8gyqa5p7q3a1khhlc5967gxq-beets-2.2.0/lib /nix/store/53z54syw8gyqa5p7q3a1khhlc5967gxq-beets-2.2.0/bin
shrinking RPATHs of ELF executables and libraries in /nix/store/wyp2xaf2yvg59f4f973sdziafpd7y2fy-beets-2.2.0-dist
checking for references to /build/ in /nix/store/wyp2xaf2yvg59f4f973sdziafpd7y2fy-beets-2.2.0-dist...
patching script interpreter paths in /nix/store/wyp2xaf2yvg59f4f973sdziafpd7y2fy-beets-2.2.0-dist
Rewriting #!/nix/store/c9m6yd8fg1flz2j5r4bif1ib5j20a0cy-python3-3.12.8/bin/python3.12 to #!/nix/store/c9m6yd8fg1flz2j5r4bif1ib5j20a0cy-python3-3.12.8
wrapping `/nix/store/53z54syw8gyqa5p7q3a1khhlc5967gxq-beets-2.2.0/bin/beet'...
Executing pythonRemoveTestsDir
Finished executing pythonRemoveTestsDir
Running phase: installCheckPhase
@nix { "action": "setPhase", "phase": "installCheckPhase" }
no Makefile or custom installCheckPhase, doing nothing
Running phase: pythonCatchConflictsPhase
@nix { "action": "setPhase", "phase": "pythonCatchConflictsPhase" }
Running phase: pythonRemoveBinBytecodePhase
@nix { "action": "setPhase", "phase": "pythonRemoveBinBytecodePhase" }
Running phase: pythonImportsCheckPhase
@nix { "action": "setPhase", "phase": "pythonImportsCheckPhase" }
Executing pythonImportsCheckPhase
Running phase: buildSphinxPhase
@nix { "action": "setPhase", "phase": "buildSphinxPhase" }
Executing buildSphinxPhase
Sphinx documentation found in docs
Executing sphinx-build with html builder
Running Sphinx v7.4.7
loading translations [en]... locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
done
making output directory... done
Converting `source_suffix = '.rst'` to `source_suffix = {'.rst': 'restructuredtext'}`.
locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
The parameter "pygment_light_style" was renamed to "pygments_light_style" (note the "s" on "pygments").
locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
building [mo]: targets for 0 po files that are out of date
writing output... 
building [html]: targets for 94 source files that are out of date
updating environment: locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
[new config] 94 added, 0 changed, 0 removed
reading sources... [  1%] changelog
reading sources... [  2%] code_of_conduct
reading sources... [  3%] contributing
reading sources... [  4%] dev/cli
reading sources... [  5%] dev/importer
reading sources... [  6%] dev/index
reading sources... [  7%] dev/library
reading sources... [  9%] dev/plugins
reading sources... [ 10%] faq
reading sources... [ 11%] guides/advanced
reading sources... [ 12%] guides/index
reading sources... [ 13%] guides/main
reading sources... [ 14%] guides/tagger
reading sources... [ 15%] index
reading sources... [ 16%] plugins/absubmit
reading sources... [ 17%] plugins/acousticbrainz
reading sources... [ 18%] plugins/advancedrewrite
reading sources... [ 19%] plugins/albumtypes
reading sources... [ 20%] plugins/aura
reading sources... [ 21%] plugins/autobpm
reading sources... [ 22%] plugins/badfiles
reading sources... [ 23%] plugins/bareasc
reading sources... [ 24%] plugins/beatport
reading sources... [ 26%] plugins/bpd
reading sources... [ 27%] plugins/bpm
reading sources... [ 28%] plugins/bpsync
reading sources... [ 29%] plugins/bucket
reading sources... [ 30%] plugins/chroma
reading sources... [ 31%] plugins/convert
reading sources... [ 32%] plugins/deezer
reading sources... [ 33%] plugins/discogs
reading sources... [ 34%] plugins/duplicates
reading sources... [ 35%] plugins/edit
reading sources... [ 36%] plugins/embedart
reading sources... [ 37%] plugins/embyupdate
reading sources... [ 38%] plugins/export
reading sources... [ 39%] plugins/fetchart
reading sources... [ 40%] plugins/filefilter
reading sources... [ 41%] plugins/fish
reading sources... [ 43%] plugins/freedesktop
reading sources... [ 44%] plugins/fromfilename
reading sources... [ 45%] plugins/ftintitle
reading sources... [ 46%] plugins/fuzzy
reading sources... [ 47%] plugins/gmusic
reading sources... [ 48%] plugins/hook
reading sources... [ 49%] plugins/ihate
reading sources... [ 50%] plugins/importadded
reading sources... [ 51%] plugins/importfeeds
reading sources... [ 52%] plugins/index
reading sources... [ 53%] plugins/info
reading sources... [ 54%] plugins/inline
reading sources... [ 55%] plugins/ipfs
reading sources... [ 56%] plugins/keyfinder
reading sources... [ 57%] plugins/kodiupdate
reading sources... [ 59%] plugins/lastgenre
reading sources... [ 60%] plugins/lastimport
reading sources... [ 61%] plugins/limit
reading sources... [ 62%] plugins/listenbrainz
reading sources... [ 63%] plugins/loadext
reading sources... [ 64%] plugins/lyrics
reading sources... [ 65%] plugins/mbcollection
reading sources... [ 66%] plugins/mbsubmit
reading sources... [ 67%] plugins/mbsync
reading sources... [ 68%] plugins/metasync
reading sources... [ 69%] plugins/missing
reading sources... [ 70%] plugins/mpdstats
reading sources... [ 71%] plugins/mpdupdate
reading sources... [ 72%] plugins/parentwork
reading sources... [ 73%] plugins/permissions
reading sources... [ 74%] plugins/play
reading sources... [ 76%] plugins/playlist
reading sources... [ 77%] plugins/plexupdate
reading sources... [ 78%] plugins/random
reading sources... [ 79%] plugins/replaygain
reading sources... [ 80%] plugins/rewrite
reading sources... [ 81%] plugins/scrub
reading sources... [ 82%] plugins/smartplaylist
reading sources... [ 83%] plugins/sonosupdate
reading sources... [ 84%] plugins/spotify
reading sources... [ 85%] plugins/subsonicplaylist
reading sources... [ 86%] plugins/subsonicupdate
reading sources... [ 87%] plugins/substitute
reading sources... [ 88%] plugins/the
reading sources... [ 89%] plugins/thumbnails
reading sources... [ 90%] plugins/types
reading sources... [ 91%] plugins/unimported
reading sources... [ 93%] plugins/web
reading sources... [ 94%] plugins/zero
reading sources... [ 95%] reference/cli
reading sources... [ 96%] reference/config
reading sources... [ 97%] reference/index
reading sources... [ 98%] reference/pathformat
reading sources... [ 99%] reference/query
reading sources... [100%] team

CONTRIBUTING.rst:385: ERROR: Unknown target name: "requests_mock".
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
copying assets... 
copying static files... done
copying extra files... done
copying assets: done
writing output... [  1%] changelog
writing output... [  2%] code_of_conduct
writing output... [  3%] contributing
writing output... [  4%] dev/cli
writing output... [  5%] dev/importer
writing output... [  6%] dev/index
writing output... [  7%] dev/library
writing output... [  9%] dev/plugins
writing output... [ 10%] faq
writing output... [ 11%] guides/advanced
writing output... [ 12%] guides/index
writing output... [ 13%] guides/main
writing output... [ 14%] guides/tagger
writing output... [ 15%] index
writing output... [ 16%] plugins/absubmit
writing output... [ 17%] plugins/acousticbrainz
writing output... [ 18%] plugins/advancedrewrite
writing output... [ 19%] plugins/albumtypes
writing output... [ 20%] plugins/aura
writing output... [ 21%] plugins/autobpm
writing output... [ 22%] plugins/badfiles
writing output... [ 23%] plugins/bareasc
writing output... [ 24%] plugins/beatport
writing output... [ 26%] plugins/bpd
writing output... [ 27%] plugins/bpm
writing output... [ 28%] plugins/bpsync
writing output... [ 29%] plugins/bucket
writing output... [ 30%] plugins/chroma
writing output... [ 31%] plugins/convert
writing output... [ 32%] plugins/deezer
writing output... [ 33%] plugins/discogs
writing output... [ 34%] plugins/duplicates
writing output... [ 35%] plugins/edit
writing output... [ 36%] plugins/embedart
writing output... [ 37%] plugins/embyupdate
writing output... [ 38%] plugins/export
writing output... [ 39%] plugins/fetchart
writing output... [ 40%] plugins/filefilter
writing output... [ 41%] plugins/fish
writing output... [ 43%] plugins/freedesktop
writing output... [ 44%] plugins/fromfilename
writing output... [ 45%] plugins/ftintitle
writing output... [ 46%] plugins/fuzzy
writing output... [ 47%] plugins/gmusic
writing output... [ 48%] plugins/hook
writing output... [ 49%] plugins/ihate
writing output... [ 50%] plugins/importadded
writing output... [ 51%] plugins/importfeeds
writing output... [ 52%] plugins/index
writing output... [ 53%] plugins/info
writing output... [ 54%] plugins/inline
writing output... [ 55%] plugins/ipfs
writing output... [ 56%] plugins/keyfinder
writing output... [ 57%] plugins/kodiupdate
writing output... [ 59%] plugins/lastgenre
writing output... [ 60%] plugins/lastimport
writing output... [ 61%] plugins/limit
writing output... [ 62%] plugins/listenbrainz
writing output... [ 63%] plugins/loadext
writing output... [ 64%] plugins/lyrics
writing output... [ 65%] plugins/mbcollection
writing output... [ 66%] plugins/mbsubmit
writing output... [ 67%] plugins/mbsync
writing output... [ 68%] plugins/metasync
writing output... [ 69%] plugins/missing
writing output... [ 70%] plugins/mpdstats
writing output... [ 71%] plugins/mpdupdate
writing output... [ 72%] plugins/parentwork
writing output... [ 73%] plugins/permissions
writing output... [ 74%] plugins/play
writing output... [ 76%] plugins/playlist
writing output... [ 77%] plugins/plexupdate
writing output... [ 78%] plugins/random
writing output... [ 79%] plugins/replaygain
writing output... [ 80%] plugins/rewrite
writing output... [ 81%] plugins/scrub
writing output... [ 82%] plugins/smartplaylist
writing output... [ 83%] plugins/sonosupdate
writing output... [ 84%] plugins/spotify
writing output... [ 85%] plugins/subsonicplaylist
writing output... [ 86%] plugins/subsonicupdate
writing output... [ 87%] plugins/substitute
writing output... [ 88%] plugins/the
writing output... [ 89%] plugins/thumbnails
writing output... [ 90%] plugins/types
writing output... [ 91%] plugins/unimported
writing output... [ 93%] plugins/web
writing output... [ 94%] plugins/zero
writing output... [ 95%] reference/cli
writing output... [ 96%] reference/config
writing output... [ 97%] reference/index
writing output... [ 98%] reference/pathformat
writing output... [ 99%] reference/query
writing output... [100%] team

generating indices... genindex done
writing additional pages... search done
copying images... [100%] plugins/beetsweb.png

dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 1 warning.

The HTML pages are in .sphinx/html/html.
Executing sphinx-build with man builder
Running Sphinx v7.4.7
loading translations [en]... locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
done
making output directory... done
Converting `source_suffix = '.rst'` to `source_suffix = {'.rst': 'restructuredtext'}`.
locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
building [mo]: targets for 0 po files that are out of date
writing output... 
building [man]: all manpages
updating environment: locale_dir /build/source/docs/locales/en/LC_MESSAGES does not exist
[new config] 94 added, 0 changed, 0 removed
reading sources... [  1%] changelog
reading sources... [  2%] code_of_conduct
reading sources... [  3%] contributing
reading sources... [  4%] dev/cli
reading sources... [  5%] dev/importer
reading sources... [  6%] dev/index
reading sources... [  7%] dev/library
reading sources... [  9%] dev/plugins
reading sources... [ 10%] faq
reading sources... [ 11%] guides/advanced
reading sources... [ 12%] guides/index
reading sources... [ 13%] guides/main
reading sources... [ 14%] guides/tagger
reading sources... [ 15%] index
reading sources... [ 16%] plugins/absubmit
reading sources... [ 17%] plugins/acousticbrainz
reading sources... [ 18%] plugins/advancedrewrite
reading sources... [ 19%] plugins/albumtypes
reading sources... [ 20%] plugins/aura
reading sources... [ 21%] plugins/autobpm
reading sources... [ 22%] plugins/badfiles
reading sources... [ 23%] plugins/bareasc
reading sources... [ 24%] plugins/beatport
reading sources... [ 26%] plugins/bpd
reading sources... [ 27%] plugins/bpm
reading sources... [ 28%] plugins/bpsync
reading sources... [ 29%] plugins/bucket
reading sources... [ 30%] plugins/chroma
reading sources... [ 31%] plugins/convert
reading sources... [ 32%] plugins/deezer
reading sources... [ 33%] plugins/discogs
reading sources... [ 34%] plugins/duplicates
reading sources... [ 35%] plugins/edit
reading sources... [ 36%] plugins/embedart
reading sources... [ 37%] plugins/embyupdate
reading sources... [ 38%] plugins/export
reading sources... [ 39%] plugins/fetchart
reading sources... [ 40%] plugins/filefilter
reading sources... [ 41%] plugins/fish
reading sources... [ 43%] plugins/freedesktop
reading sources... [ 44%] plugins/fromfilename
reading sources... [ 45%] plugins/ftintitle
reading sources... [ 46%] plugins/fuzzy
reading sources... [ 47%] plugins/gmusic
reading sources... [ 48%] plugins/hook
reading sources... [ 49%] plugins/ihate
reading sources... [ 50%] plugins/importadded
reading sources... [ 51%] plugins/importfeeds
reading sources... [ 52%] plugins/index
reading sources... [ 53%] plugins/info
reading sources... [ 54%] plugins/inline
reading sources... [ 55%] plugins/ipfs
reading sources... [ 56%] plugins/keyfinder
reading sources... [ 57%] plugins/kodiupdate
reading sources... [ 59%] plugins/lastgenre
reading sources... [ 60%] plugins/lastimport
reading sources... [ 61%] plugins/limit
reading sources... [ 62%] plugins/listenbrainz
reading sources... [ 63%] plugins/loadext
reading sources... [ 64%] plugins/lyrics
reading sources... [ 65%] plugins/mbcollection
reading sources... [ 66%] plugins/mbsubmit
reading sources... [ 67%] plugins/mbsync
reading sources... [ 68%] plugins/metasync
reading sources... [ 69%] plugins/missing
reading sources... [ 70%] plugins/mpdstats
reading sources... [ 71%] plugins/mpdupdate
reading sources... [ 72%] plugins/parentwork
reading sources... [ 73%] plugins/permissions
reading sources... [ 74%] plugins/play
reading sources... [ 76%] plugins/playlist
reading sources... [ 77%] plugins/plexupdate
reading sources... [ 78%] plugins/random
reading sources... [ 79%] plugins/replaygain
reading sources... [ 80%] plugins/rewrite
reading sources... [ 81%] plugins/scrub
reading sources... [ 82%] plugins/smartplaylist
reading sources... [ 83%] plugins/sonosupdate
reading sources... [ 84%] plugins/spotify
reading sources... [ 85%] plugins/subsonicplaylist
reading sources... [ 86%] plugins/subsonicupdate
reading sources... [ 87%] plugins/substitute
reading sources... [ 88%] plugins/the
reading sources... [ 89%] plugins/thumbnails
reading sources... [ 90%] plugins/types
reading sources... [ 91%] plugins/unimported
reading sources... [ 93%] plugins/web
reading sources... [ 94%] plugins/zero
reading sources... [ 95%] reference/cli
reading sources... [ 96%] reference/config
reading sources... [ 97%] reference/index
reading sources... [ 98%] reference/pathformat
reading sources... [ 99%] reference/query
reading sources... [100%] team

CONTRIBUTING.rst:385: ERROR: Unknown target name: "requests_mock".
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... beet.1 { } beetsconfig.5 { } done
build succeeded, 1 warning.

The manual pages are in .sphinx/man/man.
Running phase: installSphinxPhase
@nix { "action": "setPhase", "phase": "installSphinxPhase" }
Executing installSphinxPhase
Running phase: pytestCheckPhase
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.12.8, pytest-8.3.3, pluggy-1.5.0
cachedir: /tmp/pytest_cache
rootdir: /build/source
configfile: setup.cfg
plugins: cov-5.0.0, anyio-4.6.2
collecting ... 
collecting 15 items                                                            
collecting 271 items                                                           
collecting 563 items                                                           
collecting 911 items                                                           
collecting 1546 items                                                          
collected 1797 items                                                           

test/plugins/test_acousticbrainz.py ....                            [   4/1797]
test/plugins/test_advancedrewrite.py ......                         [  10/1797]
test/plugins/test_albumtypes.py .....                               [  15/1797]
test/plugins/test_art.py .......................................... [  57/1797]
....................                                                [  77/1797]
test/plugins/test_autobpm.py ..                                     [  79/1797]
test/plugins/test_bareasc.py ...                                    [  82/1797]
test/plugins/test_beatport.py ...............                       [  97/1797]
test/plugins/test_bucket.py .............F.                         [ 112/1797]
test/plugins/test_convert.py ...........................            [ 139/1797]
test/plugins/test_discogs.py ...........................            [ 166/1797]
test/plugins/test_edit.py .................                         [ 183/1797]
test/plugins/test_embedart.py ........................              [ 207/1797]
test/plugins/test_embyupdate.py ........                            [ 215/1797]
test/plugins/test_export.py ....                                    [ 219/1797]
test/plugins/test_fetchart.py .......                               [ 226/1797]
test/plugins/test_filefilter.py ........                            [ 234/1797]
test/plugins/test_ftintitle.py .......                              [ 241/1797]
test/plugins/test_hook.py .......                                   [ 248/1797]
test/plugins/test_ihate.py .                                        [ 249/1797]
test/plugins/test_importadded.py .......                            [ 256/1797]
test/plugins/test_importfeeds.py ...                                [ 259/1797]
test/plugins/test_info.py ......                                    [ 265/1797]
test/plugins/test_ipfs.py .                                         [ 266/1797]
test/plugins/test_keyfinder.py .....                                [ 271/1797]
test/plugins/test_lastgenre.py ................                     [ 287/1797]
test/plugins/test_limit.py ........                                 [ 295/1797]
test/plugins/test_lyrics.py ............ss..........ss.....         [ 326/1797]
test/plugins/test_mbsubmit.py ..                                    [ 328/1797]
test/plugins/test_mbsync.py ..                                      [ 330/1797]
test/plugins/test_mpdstats.py ...                                   [ 333/1797]
test/plugins/test_parentwork.py ssss....                            [ 341/1797]
test/plugins/test_permissions.py .......                            [ 348/1797]
test/plugins/test_play.py ............                              [ 360/1797]
test/plugins/test_player.py ....................................... [ 399/1797]
...x....x....s............xxx....sxx.xx                             [ 438/1797]
test/plugins/test_playlist.py ....................                  [ 458/1797]
test/plugins/test_plexupdate.py ...                                 [ 461/1797]
test/plugins/test_plugin_mediafield.py ......                       [ 467/1797]
test/plugins/test_random.py .                                       [ 468/1797]
test/plugins/test_replaygain.py ..........s...s.s.................. [ 503/1797]
.........                                                           [ 512/1797]
test/plugins/test_smartplaylist.py .........                        [ 521/1797]
test/plugins/test_spotify.py .....                                  [ 526/1797]
test/plugins/test_subsonicupdate.py ........                        [ 534/1797]
test/plugins/test_substitute.py ........                            [ 542/1797]
test/plugins/test_the.py .....                                      [ 547/1797]
test/plugins/test_thumbnails.py .......s.                           [ 556/1797]
test/plugins/test_types_plugin.py .......                           [ 563/1797]
test/plugins/test_web.py .........................................  [ 604/1797]
test/plugins/test_zero.py .................                         [ 621/1797]
test/test_art_resize.py .....                                       [ 626/1797]
test/test_autotag.py .............................................. [ 672/1797]
......................................                              [ 710/1797]
test/test_config_command.py ...........                             [ 721/1797]
test/test_datequery.py ................................             [ 753/1797]
test/test_dbcore.py ............................................... [ 800/1797]
.................................                                   [ 833/1797]
test/test_files.py ...ss.............................s............. [ 881/1797]
...........s...............                                         [ 908/1797]
test/test_hidden.py s.s                                             [ 911/1797]
test/test_importer.py .......s.............s.s..................... [ 956/1797]
.................................................s....s............ [1023/1797]
.........................                                           [1048/1797]
test/test_library.py ....................................ss........ [1094/1797]
................................................................... [1161/1797]
............................................................        [1221/1797]
test/test_logging.py .............                                  [1234/1797]
test/test_m3ufile.py ....s.s..                                      [1243/1797]
test/test_mb.py ................................................... [1294/1797]
....                                                                [1298/1797]
test/test_metasync.py ...                                           [1301/1797]
test/test_pipeline.py ......................                        [1323/1797]
test/test_plugins.py ........................                       [1347/1797]
test/test_query.py ................................................ [1395/1797]
................................................................... [1462/1797]
.................                                                   [1479/1797]
test/test_sort.py .................................                 [1512/1797]
test/test_template.py ............................................. [1557/1797]
..                                                                  [1559/1797]
test/test_ui.py .......................s........................... [1610/1797]
..................................................................s [1677/1797]
..............                                                      [1691/1797]
test/test_ui_commands.py .....                                      [1696/1797]
test/test_ui_importer.py ....s..................................... [1738/1797]
.............................                                       [1767/1797]
test/test_ui_init.py .....                                          [1772/1797]
test/test_util.py .s.....s...............                           [1795/1797]
test/test_vfs.py ..                                                 [1797/1797]

=================================== FAILURES ===================================
______________ BucketPluginTest.test_year_single_year_last_folder ______________

self = <test.plugins.test_bucket.BucketPluginTest testMethod=test_year_single_year_last_folder>

    def test_year_single_year_last_folder(self):
        """If a single year is given for the last bucket, extend it to current
        year."""
        self._setup_config(bucket_year=["1950", "1970"])
        assert self.plugin._tmpl_bucket("2014") == "1970"
>       assert self.plugin._tmpl_bucket("2025") == "2025"
E       AssertionError: assert '1970' == '2025'
E         
E         - 2025
E         + 1970

test/plugins/test_bucket.py:54: AssertionError
=============================== warnings summary ===============================
../../nix/store/68wyidb615gp1ixvfb4vijy3hjdvkcwj-python3.12-confuse-1.7.0/lib/python3.12/site-packages/confuse/util.py:126
  /nix/store/68wyidb615gp1ixvfb4vijy3hjdvkcwj-python3.12-confuse-1.7.0/lib/python3.12/site-packages/confuse/util.py:126: DeprecationWarning: 'pkgutil.get_loader' is deprecated and slated for removal in Python 3.14; use importlib.util.find_spec() instead
    loader = pkgutil.get_loader(name)

../../nix/store/9sqcglqn35vfivw8q1xy8g61q26kx4y0-python3.12-mediafile-0.12.0/lib/python3.12/site-packages/mediafile.py:52
  /nix/store/9sqcglqn35vfivw8q1xy8g61q26kx4y0-python3.12-mediafile-0.12.0/lib/python3.12/site-packages/mediafile.py:52: DeprecationWarning: 'imghdr' is deprecated and slated for removal in Python 3.13
    import imghdr

test/plugins/test_autobpm.py::TestAutoBPMPlugin::test_command
  /nix/store/apifxyykrnrz0xnfmcfxmzf7b2hza4ig-python3.12-librosa-0.10.2.post1/lib/python3.12/site-packages/librosa/core/intervals.py:15: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    with resources.path("librosa.core", "intervals.msgpack") as imsgpack:

test/plugins/test_autobpm.py::TestAutoBPMPlugin::test_command
  /nix/store/qr6l75w93a210965frhs5z3yq5gbrbmy-python3.12-audioread-3.0.1/lib/python3.12/site-packages/audioread/rawread.py:16: DeprecationWarning: 'aifc' is deprecated and slated for removal in Python 3.13
    import aifc

test/plugins/test_autobpm.py::TestAutoBPMPlugin::test_command
  /nix/store/qr6l75w93a210965frhs5z3yq5gbrbmy-python3.12-audioread-3.0.1/lib/python3.12/site-packages/audioread/rawread.py:17: DeprecationWarning: 'audioop' is deprecated and slated for removal in Python 3.13
    import audioop

test/plugins/test_autobpm.py::TestAutoBPMPlugin::test_command
  /nix/store/qr6l75w93a210965frhs5z3yq5gbrbmy-python3.12-audioread-3.0.1/lib/python3.12/site-packages/audioread/rawread.py:19: DeprecationWarning: 'sunau' is deprecated and slated for removal in Python 3.13
    import sunau

test/plugins/test_player.py: 69 warnings
  /nix/store/c9m6yd8fg1flz2j5r4bif1ib5j20a0cy-python3-3.12.8/lib/python3.12/multiprocessing/popen_fork.py:66: DeprecationWarning: This process (pid=684) is multi-threaded, use of fork() may lead to deadlocks in the child.
    self.pid = os.fork()

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
SKIPPED [1] test/conftest.py:12: INTEGRATION_TEST=1 required: test/plugins/test_lyrics.py::LyricsPluginSourcesTest::test_backend_sources_ok
SKIPPED [1] test/conftest.py:12: INTEGRATION_TEST=1 required: test/plugins/test_lyrics.py::LyricsPluginSourcesTest::test_google_sources_ok
SKIPPED [1] test/conftest.py:12: INTEGRATION_TEST=1 required: test/plugins/test_lyrics.py::TekstowoIntegrationTest::test_no_matching_results
SKIPPED [1] test/conftest.py:12: INTEGRATION_TEST=1 required: test/plugins/test_lyrics.py::TekstowoIntegrationTest::test_normal
SKIPPED [1] test/conftest.py:12: INTEGRATION_TEST=1 required: test/plugins/test_parentwork.py::ParentWorkIntegrationTest::test_direct_parent_work_real
SKIPPED [1] test/conftest.py:12: INTEGRATION_TEST=1 required: test/plugins/test_parentwork.py::ParentWorkIntegrationTest::test_force_real
SKIPPED [1] test/conftest.py:12: INTEGRATION_TEST=1 required: test/plugins/test_parentwork.py::ParentWorkIntegrationTest::test_no_force_real
SKIPPED [1] test/conftest.py:12: INTEGRATION_TEST=1 required: test/plugins/test_parentwork.py::ParentWorkIntegrationTest::test_normal_case_real
SKIPPED [1] test/plugins/test_player.py:941: Skipped
SKIPPED [1] test/plugins/test_player.py:1131: Skipped
SKIPPED [1] test/plugins/test_replaygain.py:201: r128 tags for opus not supported on backend command
SKIPPED [1] test/plugins/test_replaygain.py:263: r128 tags for opus not supported on backend command
SKIPPED [1] test/plugins/test_replaygain.py:299: r128 tags for opus not supported on backend command
SKIPPED [1] test/plugins/test_thumbnails.py:263: GIO library not found
SKIPPED [1] test/test_files.py:97: no reflink support for libdir
SKIPPED [1] test/test_files.py:102: no reflink support for libdir
SKIPPED [1] test/test_files.py:286: no reflink support for libdir
SKIPPED [1] test/test_files.py:571: no reflink support for libdir
SKIPPED [1] test/test_hidden.py:32: sys.platform is not darwin
SKIPPED [1] test/test_hidden.py:49: sys.platform is not windows
SKIPPED [2] test/test_importer.py:213: no reflink support for libdir
SKIPPED [1] test/test_importer.py:269: unrar program not found
SKIPPED [1] test/test_importer.py:269: Implement me!
SKIPPED [1] test/test_importer.py:1192: write me
SKIPPED [1] test/test_importer.py:1295: write me
SKIPPED [1] test/test_library.py:476: unimplemented: #359
SKIPPED [1] test/test_library.py:487: unimplemented: #359
SKIPPED [1] test/test_m3ufile.py:119: win32
SKIPPED [1] test/test_m3ufile.py:71: win32
SKIPPED [1] test/test_ui.py:355: not yet implemented
SKIPPED [1] test/test_ui.py:1401: bash-completion script not found
SKIPPED [1] test/test_util.py:135: fs is case sensitive
SKIPPED [1] test/test_util.py:94: unimplemented: #359
XFAIL test/plugins/test_player.py::BPDControlTest::test_implements_control - reason: 
XFAIL test/plugins/test_player.py::BPDQueueTest::test_implements_queue - reason: 
XFAIL test/plugins/test_player.py::BPDDatabaseTest::test_implements_database - reason: 
XFAIL test/plugins/test_player.py::BPDMountsTest::test_implements_mounts - reason: 
XFAIL test/plugins/test_player.py::BPDStickerTest::test_implements_stickers - reason: 
XFAIL test/plugins/test_player.py::BPDPartitionTest::test_implements_partitions - reason: 
XFAIL test/plugins/test_player.py::BPDDeviceTest::test_implements_devices - reason: 
XFAIL test/plugins/test_player.py::BPDReflectionTest::test_implements_reflection - reason: 
XFAIL test/plugins/test_player.py::BPDPeersTest::test_implements_peers - reason: 
FAILED test/plugins/test_bucket.py::BucketPluginTest::test_year_single_year_last_folder - AssertionError: assert '1970' == '2025'
= 1 failed, 1753 passed, 34 skipped, 9 xfailed, 75 warnings in 93.14s (0:01:33) =
```

</details>

## Additional context

The test `test/plugins/test_bucket.py::BucketPluginTest::test_year_single_year_last_folder` fails. We could just disable the test, but maybe someone is able to find out _why_ that test fails.
<!-- Add any other context about the problem here. -->

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.7, NixOS, 25.05 (Warbler), 25.05.20250102.3da6bd3`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.92.0-dev-pre20241230-5c7ea4f
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/anton/.config/nix/nix.conf:/nix/store/zs83w398qva6syb97mb18i7mbas33a89-stylix-kde-config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/anton/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/anton/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/anton/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/8gqhnjl4l26ksa02arw7k5axiip9by9b-lix-2.92.0-dev-pre20241230-5c7ea4f/share`
 - channels(root): `""`
 - nixpkgs: `/etc/nix/inputs/nixpkgs`


## Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->
@doronbehar 

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/tools/audio/beets/default.nix
+++ b/pkgs/tools/audio/beets/default.nix
@@ -3,6 +3,7 @@
   callPackage,
   fetchFromGitHub,
   python3Packages,
+  fetchpatch,
 }:
 /*
   ** To customize the enabled beets plugins, use the pluginOverrides input to the
@@ -17,11 +18,22 @@
   **   alternatives = { enable = true; propagatedBuildInputs = [ beetsPackages.alternatives ]; };
   ** }; }
 */
+let
+  extraPatches = [
+    # Bash completion fix for Nix
+    ./patches/bash-completion-always-print.patch
+    # Remove after next release.
+    (fetchpatch {
+      url = "https://github.com/beetbox/beets/commit/bcc79a5b09225050ce7c88f63dfa56f49f8782a8.patch?full_index=1";
+      hash = "sha256-Y2Q5Co3UlDGKuzfxUvdUY3rSMNpsBoDW03ZWZOfzp3Y=";
+    })
+  ];
+in
 lib.makeExtensible (self: {
   beets = self.beets-stable;
 
   beets-stable = callPackage ./common.nix rec {
-    inherit python3Packages;
+    inherit python3Packages extraPatches;
     version = "2.2.0";
     src = fetchFromGitHub {
       owner = "beetbox";
@@ -29,16 +41,12 @@ lib.makeExtensible (self: {
       rev = "v${version}";
       hash = "sha256-jhwXRgUUQJgQ/PLwvY1UfHCJ9UC8DcdBpE/janao0RM=";
     };
-    extraPatches = [
-      # Bash completion fix for Nix
-      ./patches/bash-completion-always-print.patch
-    ];
   };
 
   beets-minimal = self.beets.override { disableAllPlugins = true; };
 
   beets-unstable = callPackage ./common.nix {
-    inherit python3Packages;
+    inherit python3Packages extraPatches;
     version = "2.2.0-unstable-2024-12-02";
     src = fetchFromGitHub {
       owner = "beetbox";
@@ -46,10 +54,6 @@ lib.makeExtensible (self: {
       rev = "f92c0ec8b14fbd59e58374fd123563123aef197b";
       hash = "sha256-jhwXRgUUQJgQ/PLwvY1UfHCJ9UC8DcdBpE/janao0RM=";
     };
-    extraPatches = [
-      # Bash completion fix for Nix
-      ./patches/bash-completion-always-print.patch
-    ];
   };
 
   alternatives = callPackage ./plugins/alternatives.nix { beets = self.beets-minimal; };

